compiler/Pure.ml

```diff
@@ -397,15 +397,14 @@ type predicates = { trait_type_constraints : trait_type_constraint list }
 
 type type_decl = {
   def_id : TypeDeclId.id;
-  is_local : bool;
   llbc_name : llbc_name;
       (** The original name coming from the LLBC declaration *)
```

Comment:
This is not necessary anymore, because the name is now stored in the `item_meta` field (for which we should add a comment). Same for the other declarations.

Reply:
That's not true because of renames. Or do you want renames to update `item_meta`?

Reply:
The LLBC name is kept inside the `item_meta`, together with the `rename` attribute. When we generate the name for the extracted definitions we combine the LLBC name (today, from this field, but could be from the `item_meta`) with the content of the rename attribute, if there is.

Reply:
Oh, I thought that was done earlier in the pipeline. Makes sense to remove the field then

Reply:
Done here: https://github.com/AeneasVerif/aeneas/pull/272